### ISSUE TITLE
Support "math" IA2 object attribute in Chrome and Firefox 

### ISSUE THREAD

**Issue by aleventhal** [nvaccess/nvda#17296]
Currently NVDA supports math on ROLE_SYSTEM_EQUATION objects by using ISimpleDOMNode::get_innerHtml().
In the interests of eventually being able to deprecate and remove ISimpleDOMNode support, NVDA should first check whether the "math" object attribute exists, and only use ISimpleDOMNode as a fallback.
Chromium has added support for this in recent Canary builds.
Here's a issue entry for this to also be supported by Mozilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1924832

**Comment by NSoiffer** [comment]
Is this related to [NVDA cannot navigate math content in Chromium](https://github.com/nvaccess/nvda/issues/17421) ?

**Comment by aleventhal** [comment]
It didn't cause the problem, but the change that caused the problem is under the same project -- eventually getting rid of ISimpleDOMNode.

**Comment by Adriani90** [comment]
This is related or even a duplicate of #13702.

**Comment by Adriani90** [comment]
cc: @SaschaCowley
